pulp_container/app/authorization.py

```diff
@@ -31,19 +31,19 @@ class AuthorizationService:
     according to a user's scope.
     """
 
-    def __init__(self, user, service, scope):
+    def __init__(self, user, service, scopes):
         """
         Store class-wide variables and initialize a dictionary used for determining permissions.
 
         Args:
             user (django.contrib.auth.models.User): Requesting user.
             service (str): Name of the service access is granted to.
-            scope (str): Scope of the resource that is to be accessed.
+            scopes (list): Scope of the resource that is to be accessed.
```

Comment:
Scope(s) of the resource(s) that is to be accessed.

Reply:
I think this change was meant to be requested here:
```suggestion
            scopes (list): Scopes of the resources that is to be accessed.
```

Reply:
Just realizing now: Scopes of the resources that _are_ to be accessed.